cli/command/task_test.go

```diff
@@ -182,34 +146,16 @@ func TestReadTaskLogs(t *testing.T) {
 		ReadTaskLogs("tsk_id", query).
 		Return([]models.LogFile{}, nil)
 
-	flags := map[string]interface{}{
-		"tail":  100,
-		"start": "start",
-		"end":   "end",
-	}
-
-	c := testutils.NewTestContext(t, []string{"tsk_name"}, flags)
-	if err := command.logs(c); err != nil {
+	input := "l0 task logs --tail 100 --start start --end end tsk_name"
```

Comment:
Should all flags be delimitated?
```
input := "l0 task logs "
input += "--tail 100 "
input += "--start start "
input += "--end end tsk_name"
```

Reply:
for like anytime there are flags? I could get behind that for consistency's sake

Reply:
Yup.